esp32-hal/.cargo/config.toml

```diff
@@ -3,8 +3,13 @@ runner = "espflash flash --monitor"
 
 [build]
 rustflags = [
-  "-C", "link-arg=-nostartfiles",
-  "-C", "link-arg=-Wl,-Tlinkall.x",
+  # GCC
```

Comment:
shouldn't we keep GCC as the default for now?

Reply:
Oops, yes :D